### ISSUE TITLE
Wildcard allowances don't permit all as the documentation states it should.

### ISSUE THREAD

**Issue by lanmaster53** [bprinty/Flask-Authorize#31]
Setting the default `__allowances__` value to `*` for roles doesn't result in access to everything as the documentation states. Rather, it results in the following error because the `user_is_allowed` method never checks check for the `*` value, or the `default_allowances` method fails to convert the `*` to something useful.

```
  File "/usr/local/lib/python3.7/site-packages/flask_authorize/plugin.py", line 208, in user_is_allowed
    allowances.extend(cred.allowances.get(key, default))
AttributeError: 'str' object has no attribute 'get'
```

**Comment by bprinty** [comment]
Are you using the most recent version `0.1.8` on PyPI? The previous version didn't support the `__allowances__ = '*'` syntax.

**Comment by bprinty** [comment]
You can `pip install -U Flask-Authorize` to make sure you have the most updated version (I updated it earlier today).

**Comment by lanmaster53** [comment]
> Are you using the most recent version `0.1.8` on PyPI? The previous version didn't support the `__allowances__ = '*'` syntax.

Yes, I made sure I had the most recent one. You check for `*` in restrictions, but not in allowances. You wrote all the code to support the fail-safe piece, but I think you left out some logic for the opposite. I tested it thoroughly because I needed a role that defaulted to everything and this stopped me dead in my tracks. I didn't look close at restrictions to see if the more permissive option was working there, but it might have been overlooked as well.
